config/helm/chart/default/templates/Common/operator/clusterrole-operator.yaml

```diff
@@ -21,15 +21,15 @@ rules:
   - apiGroups:
       - ""
     resources:
-      - nodes
+      - nodes # Required by nodes controller for node cache and mark for termination handling
```

Comment:
I really like the comments, I am just unsure if we should keep them here. A different place outside the actual code would feel better to me tbh. WDYT?